docs/dev/cut-a-new-release.md

```diff
@@ -1,38 +1,61 @@
 # Glean Release Process
 
-> **Important:** The Glean SDK does not yet have a fully streamlined release process. All information here is preliminary.
-> At this point the release process only produces release artifacts for the Kotlin library.
-> The Rust crate is not released on its own. No artifacts for iOS are generated.
-
 These are the steps needed to cut a new release from latest master.
 
 1. Update the changelog.
-    1. In `CHANGELOG.md`:
-        1. Replace `# Unreleased Changes` with `# v<new-version-number> (_<current date>_)`.
-        2. Replace `master` in the Full Changelog link to be `v<new-version-number>`. E.g. if you are releasing 0.1.0, the link should be
-            ```
-            [Full Changelog](https://github.com/mozilla/glean/compare/v0.0.1...v0.1.0)
-            ```
-            Note that this needs three dots (`...`) between the two tags (two dots is different). Yes, the second tag doesn't exist yet, you'll make it later.
-        3. Optionally, go over the commits between the past release and this one and see if anything is worth including.
-        4. Make sure the changelog follows the format of the other changelog entries.
-            - Note that we try to provide PR or issue numbers (and links) for each change. Please add these if they are missing.
-
-    2. Add a new `# Unreleased Changes` on top
-
-2. Bump the versions
-    * Rust crates: Bump `version` in [`glean-core/Cargo.toml`](https://github.com/mozilla/glean/blob/master/glean-core/Cargo.toml) and [`glean-core/ffi/Cargo.toml`](https://github.com/mozilla/glean/blob/master/glean-core/ffi/Cargo.toml).  **NOTE:** There are two versions to update in `glean-core/ffi/Cargo.toml`.
-    * Kotlin package: Bump `libraryVersion` in the top-level [.buildconfig.yml](https://github.com/mozilla/glean/blob/master/.buildconfig.yml) file.
-    * Gradle plugin: Bump `project.ext.gleanVersion` in [`GleanGradlePlugin.groovy`](https://github.com/mozilla/glean/blob/master/gradle-plugin/src/main/groovy/mozilla/telemetry/glean-gradle-plugin/GleanGradlePlugin.groovy).
-    * Be sure you're following semver, and if in doubt, ask.
-    * Run `make build` to make sure the `Cargo.lock` file is updated.
```

Comment:
Just remembered: We'll also need to automate this step.

Reply:
I saw that, removed it and then forgot to add it to the script